### DISCUSSION TITLE
Personal Porfolio / Pirate ship journey

### DISCUSSION THREAD

**Discussion by andreatoffanello** [enable3d/enable3d#337] [Show and tell]
Hi there, 
I used enable3d for the ship moving and the collision with the islands for my personal website.
https://andreatoffanello.com
Let me know your impressions

<img width="1721" alt="Schermata 2022-10-30 alle 16 57 37" src="https://user-images.githubusercontent.com/3775281/198888545-76ffe276-73dd-401d-8102-e0d6c37cb212.png">

**Comment by yandeu** [comment]
Nice!! Really love the movement of the ship.
Would you like to put it on the enable3d website?

**Reply by andreatoffanello** [reply]
Yeah it would be great!

**Reply by yandeu** [reply]
Like this?
https://enable3d.io/#projects

**Reply by scenaristeur** [reply]
@yandeu you could add flipball to projects
https://scenaristeur.github.io/flipball/
Vuejs +enable3d, with a selector for Json file describing a table -> a mobile pinball hosted on gh-pages installable as a mobile app.

1select flipball table
2 tap on top to launch the ball
3 tap on left bottom or right bottom to play 
Could be used on non touch devices too with keyboard arrow ( up, left, right)

**Reply by yandeu** [reply]
The Marble and Modele do not seem to work. The Flipball works great but is cut off at the bottom. Might want to fix it first?

**Reply by scenaristeur** [reply]
@yandeu perhaps not polute @andreatoffanello  discussion and could switch to https://github.com/enable3d/enable3d/discussions/241 to talk about flipball

**Comment by scenaristeur** [comment]
@andreatoffanello perharps links as blank opening to another tab, because when i clicked on your linkedin at the end , it opens in the same page, and replace the journey, so when when I want to come back your other link, i have to restart the journey from beginning ;-(

**Reply by andreatoffanello** [reply]
You're damn right... thank you!

**Comment by walaeddine-riahi** [comment]
@andreatoffanello can you share the code please??

**Comment by Richardengle** [comment]
the look of this game is so cool.  im having trouble with hitting some rocks in the open sea tho, lol.
